packages/edit-site/src/components/dataviews/style.scss

```diff
@@ -129,3 +129,25 @@
 .dataviews-action-modal {
 	z-index: z-index(".dataviews-action-modal");
 }
+
+.dataviews-bulk-actions-popover .components-popover__content {
+	min-width: max-content;
+}
+
+.dataviews-bulk-actions-toolbar-wrapper {
+	display: flex;
+	flex-grow: 1;
+	width: 100%;
+}
+
+.dataviews-table-view__selection-column label {
```

Comment:
Note the related comment above, this could be `th [data-field-id="dataviews-selection"]`.